File: Netbeans/Java/Lab6/test/model/ReversedTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author prostrmk
 */
public class ReversedTest {
    
    public ReversedTest() {
    }

    @Test
    public void ReversedTest() {
        int a = 321;
        assertEquals(123, Reversed.reverser(a));
        
    }
    
}

File: Csharp/Laba6/Laba6/TestFolder/ReversedTest.cs
using NUnit.Framework;
using System;

namespace Laba6
{
	[TestFixture ()]
	public class ReversedTest
	{
		[Test ()]
		public void ReverserTest ()
		{
			int rev = 12345;
			Assert.AreEqual (54321, Reversed.Reverser (rev));
		}
	}
}

